forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java

```diff
@@ -207,6 +207,36 @@ public void testAccessToken() throws Exception {
         assertNotNull(accessToken.getValue());
     }
 
+    @Test
+    public void testRevokeAccessToken() throws Exception {
+        frUserHappyPath();
+        
+        //Revoke the token and check the listener onSuccess and onFailure methods
+        FRUser.getCurrentUser().revokeAccessToken(new FRListener<Void>() {
+            @Override
+            public void onSuccess(Void result) {
+                assertNotNull(FRUser.getCurrentUser());
+                assertTokenRevoked();
+            }
+
+            @Override
+            public void onException(Exception e) {
+                assertNotNull(FRUser.getCurrentUser());
+                assertTokenRevoked();
+            }
+        });
+    }
+
+    private void assertTokenRevoked() {
+        try {
+            AccessToken accessToken = FRUser.getCurrentUser().getAccessToken();
```

Comment:
I don't think using FRUser.getCurrentUser().getAccessToken() will return null, it will always retry to get an accesstoken, if unable to get an accesstoken, it will throw exception.

To valid the token is actually clear, u can check the storage directly, since it is using Config.getInstance().setSharedPreferences(context.getSharedPreferences(DEFAULT_TOKEN_MANAGER_TEST, Context.MODE_PRIVATE)); you can check if that sharedpreferences has any data.

Reply:
Thanks will have a look on that